src/spaceObjects/spaceship.cpp

```diff
@@ -466,7 +468,12 @@ void SpaceShip::update(float delta)
             warp_request = 0.0;
     }
 
-    float rotationDiff = sf::angleDifference(getRotation(), target_rotation);
+    float rotationDiff;
+    if (fabs(turnSpeed) < 0.0005f) {
+        rotationDiff = sf::angleDifference(getRotation(), target_rotation);
+    } else {
+        rotationDiff = turnSpeed;
+    }
```

Comment:
Shouldn't we update the target_rotation here somewhere, else when the turnSpeed goes to 0.0 the ship will rotate back to the target_rotation.

Reply:
currently the only place that sets value to `turnSpeed` is the execution of `CMD_TURN_SPEED`, which also resets `target_rotation` to the current ship's rotation. setting it here seems redundant but mostly harmless.

Reply:
You're right, because the turnSpeed is only updated that way, and thus can only be set to 0 that way. Didn't think of that. That part still feels a bit iffy tough.